### PR TITLE
Fix : Section filters and products should not be highlighted during search in wire section [NHUB-320]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: newsroom/search/service.py [hunk 1]
```diff
@@ -514,37 +513,45 @@ def prefill_search_items(self, search):
...
-                    "description_text": field_settings,
-                    "definition_short": field_settings,
-                    "name": field_settings,
-                    "definition_long": field_settings,
+            for field in fields_to_highlight:
+                search.source["highlight"]["fields"][field] = {
+                    "number_of_fragments": 0,
+                    "highlight_query": {
+                        "query_string": {
+                            "query": query_string or advanced_search.get("all") or advanced_search.get("any"),
```
[the remark this answers is not in the document]
Comment (devketanpro) [reply newsroom/search/service.py]: ok

File: newsroom/search/service.py [hunk 2]
```diff
@@ -511,21 +511,33 @@ def prefill_search_items(self, search):
...
+                should_queries.append(query_string(query))
+            all_query = advanced_search.get("all")
+            if all_query:
+                should_queries.append(query_string(all_query))
+            any_query = advanced_search.get("any")
+            if any_query:
+                should_queries.append(query_string(any_query))
+
+            search.query.setdefault("bool", {})
+            search.query["bool"]["should"] = should_queries
```
Comment (petrjasek) [inline newsroom/search/service.py:533]: there is a helper to create such query, would be better to use that one:
https://github.com/superdesk/newsroom-core/blob/develop/newsroom/search/service.py#L708

Comment (devketanpro) [reply newsroom/search/service.py:533]: Ok I'll check

File: newsroom/search/service.py [hunk 3]
```diff
@@ -550,14 +550,16 @@ def prefill_search_highlights(self, search, req):
...
-                    "highlight_query": {"bool": {"should": should_queries}},
+                    "highlight_query": {"bool": {"filter": highlight_search.query["bool"]["filter"]}},
                 }
 
-            search.source["highlight"]["pre_tags"] = ['<span class="es-highlight">']
-            search.source["highlight"]["post_tags"] = ["</span>"]
-            search.source["highlight"]["require_field_match"] = False
+            highlight_search.source["highlight"]["pre_tags"] = ['<span class="es-highlight">']
+            highlight_search.source["highlight"]["post_tags"] = ["</span>"]
+            highlight_search.source["highlight"]["require_field_match"] = False
```
Comment (petrjasek) [inline newsroom/search/service.py:560]: not sure if we need this, it should probably only highlight those requested fields

**Review by petrjasek — APPROVED** [review]

File: newsroom/search/service.py (hunk 3 above)
Comment (devketanpro) [reply newsroom/search/service.py]: yep

**Review by petrjasek — APPROVED** [review]

**Pull request merged**